### PR TITLE
Pr notapatch rubocop rspec

### PR DESCRIPTION
**Pull request by notapatch**
Updating rubocop rspec

### CODE REVIEW

File: spec/models/additional_document_validation_request_spec.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
   it_behaves_like "ValidationRequest", described_class, "additional_document_validation_request"
 
   it_behaves_like("Auditable") do
-    let(:subject) { create(:additional_document_validation_request) }
+    subject { create(:additional_document_validation_request) }
```
Comment (allbecauseyoutoldmeso) [inline spec/models/additional_document_validation_request_spec.rb:11]: I like!

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -310,3 +310,14 @@ Layout/LineLength:
     - 'spec/system/planning_applications/validating_spec.rb'
     - 'db/migrate/20211020153256_create_delayed_jobs.rb'
     - 'db/migrate/20211122103943_update_enquiries_paragraph.rb'
+
+Style/OpenStructUse:
```
Comment (allbecauseyoutoldmeso) [inline .rubocop_todo.yml]: Hmm. I think this is okay, but let's find time to remove all the `OpenStructs` at some point!

I have a personal wish-list of refactors for between now and January 😆

File: spec/support/matchers.rb [hunk 3]
```diff
@@ -29,3 +29,5 @@
 RSpec::Matchers.define(:have_target_id) do |target_id|
   match { |url| URI.parse(url).fragment == target_id }
 end
+
+RSpec::Matchers.define_negated_matcher :not_change, :change
```
Comment (allbecauseyoutoldmeso) [inline spec/support/matchers.rb:33]: Niiice!

File: spec/components/proposal_details/summary_component_spec.rb [hunk 4]
```diff
@@ -35,15 +35,15 @@
   describe "#auto_answered?" do
     context "when question was auto answered" do
       it "returns true" do
-        expect(summary_component.send(:auto_answered?)).to eq(true)
+        expect(summary_component.send(:auto_answered?)).to be(true)
```
Comment (allbecauseyoutoldmeso) [inline spec/components/proposal_details/summary_component_spec.rb:38]: I didn't know about `be` until just the other day. Much more human-readable 🤩

File: spec/controllers/additional_document_validation_requests_controller_spec.rb [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 require "rails_helper"
 
-RSpec.describe AdditionalDocumentValidationRequestsController, type: :controller do
+RSpec.describe AdditionalDocumentValidationRequestsController do
```
Comment (allbecauseyoutoldmeso) [inline spec/controllers/additional_document_validation_requests_controller_spec.rb:5]: I didn't know we could do this! So much nicer 😌

File: spec/factories/additional_document_validation_requests.rb [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 FactoryBot.define do
   factory :additional_document_validation_request do
-    planning_application { create :planning_application, :invalidated }
+    planning_application { create(:planning_application, :invalidated) }
```
Comment (allbecauseyoutoldmeso) [inline spec/factories/additional_document_validation_requests.rb:5]: 🌟

Comment (allbecauseyoutoldmeso) [reply spec/factories/additional_document_validation_requests.rb:5]: I guess either way is fine, but personally I prefer this option 😆

**Review by allbecauseyoutoldmeso — APPROVED** [review]
So much awesome 🎉 🎉 🎉 🎉 🎉 🎉 🎉

File: spec/components/proposal_details/summary_component_spec.rb (hunk 4 above)
Comment (EGiataganas) [reply spec/components/proposal_details/summary_component_spec.rb:38]: Yeah I totally agree, I also prefer the `be` compare to `eq` :D

**Review by EGiataganas — APPROVED** [review]

**Pull request merged**